index.html

```diff
@@ -180,30 +180,33 @@
       </div>
       <div class="row">
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Install Bazel</h3>
-          <p>See our <a href="docs/install.html">Installation Guide</a> to learn
-             how to download and install Bazel.
+          <h3>Java</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/java.html">Build Java tutorial</a>.
           </p>
         </div>
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Run Bazel</h3>
-          <p>See our
-             <a href="docs/getting-started.html">Getting Started Guide</a> to
-             learn how to write a BUILD file and run Bazel.
+          <h3>C++</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/cpp.html">Build C++ tutorial</a>.
           </p>
         </div>
       </div>
       <div class="row">
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Bazel Commands</h3>
-          <p>See our <a href="docs/bazel-user-manual.html">User Manual</a> to
-             learn about the Bazel command-line tool.
+          <h3>Android</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/app.html">mobile app tutorial</a>.
           </p>
         </div>
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Learn More</h3>
-          <p>Take a look at our <a href="docs/install.html">documentation</a>
-             and read our <a href="faq.html">FAQ</a>.
+          <h3>iOS</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/app.html">mobile app tutorial</a>.
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-4 col-sm-offset-4">
+          <h3>Other languages</h3>
+          <!-- Link to GitHub until we have a better {{ site.docs_site_url }}/be/overview.html-->
```

Comment:
Ok I am goign to delte this to make progress.
But do not forget: we are not talking about removing this overview. We are just talking about not linking it from the home page.
Damien, when you access it, I bet you are never using the link at the bottom of the homepage, the reason is simple: it is not the same user journey.

Also, let's track all this feedback as GH issue, not in this PR.
iOS tutorial indeed needs a rewrite with apple_rules, I added a comment to https://github.com/bazelbuild/bazel/issues/3245